code/modules/orbits/spaceship.dm

```diff
@@ -44,6 +49,7 @@ GLOBAL_VAR_INIT(current_orbit,STANDARD_ORBIT)
 /obj/machinery/computer/navigation/Initialize() //need anything special?
 	. = ..()
 	desc = "The navigation console for the [SSmapping.configs[SHIP_MAP].map_name]."
+	addtimer(VARSET_CALLBACK(src, changing_orbit, FALSE), 1 HOURS) //people running away far too quickly it seems
```

Comment:
1 hours is a lot 30 minutes would be better

Reply:
Sure